src/app/components/age-selector/age-selector.component.html

```diff
@@ -6,7 +6,7 @@
       <ng5-slider class="slider" [options]="options" [(value)]="lowValue" [(highValue)]="highValue"
         (userChangeEnd)="sliderValueChanged()">
       </ng5-slider>
-      <div class="label max">{{ options.ceil }}</div>
+      <div class="label max">89+</div>
```

Comment:
Don't hard code this. Simple add a '+' sign after the interpolation, and change interpolation to {{ options.ceil - 1 }}